ui/packages/consul-ui/app/components/topology-metrics/popover/index.hbs

```diff
@@ -4,22 +4,21 @@
 >
 
 {{#if (eq @type 'deny')}}
-
   <InformedAction
     class="dangerous"
     {{did-insert (set this 'popover')}}
   >
     <:header>
       <h3>
-        Connection Denied
+        {{t "components.consul.topology-metrics.popover.deny.header"}}
       </h3>
     </:header>
     <:body>
       <p>
         {{#if @item.Intention.HasExact}}
-          Change the action of this intention to allow.
+          {{t "components.consul.topology-metrics.popover.deny.body.isExact"}}
         {{else}}
-          Add an intention that allows these two services to connect.
+          {{t "components.consul.topology-metrics.popover.deny.body.notExact"}}
         {{/if}}
       </p>
```

Comment:
I couldn't reply to the previous comment about `<p>` tags as the code has gone now, so just moving it here:

> In this scenario I just have 1 <p>. Use still want to use default-allow.body.p0?

In this case:

```hbs
<:body>
  {{t "body" htmlSafe=true}}
</:body>
```

```yaml
body: |
  <p>Paragraph one</p>
  <p>Paragraph <em>two</em></p>
```

At some point in the future if we can do:

```yaml
body: |
  Paragraph one
  Paragraph _two_
```

..and shim a markdown parser into the `t` helper, that would be much better - but that's probably something for a later date.